src/include/mom_func.h

```diff
@@ -325,6 +325,7 @@ enum stagefile_errcode {
 struct copy_info {
 	pbs_list_link al_link;		/* link to all copy info list */
 	char *jobid;			/* job id to which this info belongs */
+	job *pjob;			/* pointer to job structure */
```

Comment:
Good point! I haven't thought/try that, I will try and if works will remove jobid.

Reply:
@subhasisb I looked into this, and it looks like pjob can be null (as per `post_cpyfile` and `req_cpyfile`, windows version) but jobid can't be null as we copy it from `preq->rq_ind.rq_cpyfile.rq_jobid` in `req_cpyfile`. Now I am not able to recollect when pjob will be null, from code its looks like it will stage-in case but still not sure when pjob will be null.